book/src/user/elasticsearch.md

```diff
@@ -0,0 +1,101 @@
+# Kibana blockchain explorer
+
+The goal here is to export block data from Zebra into an [elasticsearch](https://www.elastic.co/) database and visualize it with the [kibana](https://www.elastic.co/kibana/) front end. 
+
+**Attention:** This is an experimental feature tested only in the Zcash Testnet.
+
+Elasticsearch support for zebra was introduced to Zebra in [pull request #6274](https://github.com/ZcashFoundation/zebra/pull/6274).
+
+## Download, build and run Elasticsearch
+
+Installing elasticsearch is easy in linux and macOS by following the [.tar.gz installation guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/targz.html).
+
+Make sure you end up with an `elasticsearch` binary. Run it:
+
+```sh
+./bin/elasticsearch
+```
+
+The first time you run the database, `elastic` password and the enrollment token for Kibana will displayed in the screen (See [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/targz.html#targz-running)). Please save these as you will need them.
+
+Elasticsearch will listen in https://localhost:9200 by default.
+
+## Download, build and run Kibana
+
+Installing kibana is also easy in linux and macOS following the [.tar.gz installation guide](https://www.elastic.co/guide/en/kibana/current/targz.html).
+
+
+Make sure you end up with a `kibana` binary and run it:
+
+```sh
+./bin/kibana
+```
+
+ The first time you run kibana, it will provide a link for configuration:
+ 
+```sh
+Kibana has not been configured.
+    
+Go to http://localhost:5601/?code=405316 to get started.
+```
+
+Visit the url to get started, you will need the kibana enrollment token from elasticsearch and the elastic password from the previous step.
+
+Kibana will listen in https://localhost:5601 by default.
+
+You are now ready to start bumping data into elasticsearch with Zebra.
+
+## Download and build zebra with `elasticsearch` feature
+
+Elasticsearch is an optional and experimental feature, we need to build and install with the `elasticsearch` rust feature enabled using the following command:
+
+```sh
+cargo install --features elasticsearch --locked --git https://github.com/ZcashFoundation/zebra zebrad
```

Comment:
This command will be true only when https://github.com/ZcashFoundation/zebra/pull/6274 gets merged.